lib/cocoapods/validator.rb

```diff
@@ -857,6 +857,10 @@ def xcodebuild(action, scheme, configuration)
         command += Fourflusher::SimControl.new.destination(:oldest, 'tvOS', deployment_target)
       end
 
+      if config.verbose
+        command += ' -verbose'
```

Comment:
nit: I think this can be `command += ' -verbose' if config.verbose?`

Also use `verbose?` with question mark instead.

Reply:
Thanks, Ruby's not exactly my specialty :)